Musicplayer.css

```diff
@@ -92,14 +101,17 @@ ul li ul li:hover{background: black}
 }
 
 .footer{
-	background-color: black;
+	background-color: #0C59CF;
```

Comment:
21/9: I create a footer and apply all the element that I decided to use like the fonts, color with some other small function like position of the text.